types/index.d.ts

```diff
@@ -33,8 +31,14 @@ type VueTestUtilsRenderOptions = Omit<
   'attachTo' | 'shallow' | 'propsData'
 >
 type VueTestingLibraryRenderOptions = {
-  store?: StoreOptions<{}>
-  routes?: RouteRecordRaw[]
+  /**
+   * @deprecated Use `global.plugins` array instead.
+   */
+  store: Record<string, any>
+  /**
+   * @deprecated Use `global.plugins` array instead.
+   */
+  routes?: Record<string, any>
```

Comment:
I think the `@deprecated` notice is good. The types seem a little weird though -- particularly `routes`.  It's technically not the real type. The imports could be kept to ensure the correct types appear... But at the same time it doesn't matter too much since it can't be used? What are your thoughts?

Reply:
Actually I was tempted to use `any`, as it won't really work and user should be replacing this. Not entirely sure why I ended up setting `Record<string, any>`, looks like a leftover 😂 I think I'll leave it as `any` because, as you mention, they can't be used and are only defined here so they can be marked as `@deprecated`. Thanks!